recipes/idna-ssl/meta.yaml

```diff
@@ -13,15 +13,15 @@ source:
 
 build:
   number: 0
+  noarch: python
   script: python -m pip install --no-deps --ignore-installed .
-  skip: True  # [not (py35 or py36)]
 
 requirements:
   build:
-    - python
+    - python >=3.5,<3.7
```

Comment:
why less than 3.7?

Reply:
Because of [this issue is fixed in 3.7](https://github.com/aio-libs/idna-ssl#motivation)

Reply:
that would do it!